samples/unit-tests/axis/stacklabels/demo.js

```diff
@@ -324,3 +324,79 @@ QUnit.test('StackLabels outside xAxis min & max range are displayed #12294', fun
         'This stack-label text x attribute should be equal to set padding #12308'
     );
 });
+
+QUnit.test('StackLabels Initial animation - defer test #12901', function (assert) {
+
+    var clock = null;
+
+    try {
+        clock = TestUtilities.lolexInstall();
+
+        var chart = Highcharts.chart('container', {
+                chart: {
+                    type: 'column'
+                },
+
+                plotOptions: {
+                    series: {
+                        stacking: 'normal',
+                        animation: {
+                            defer: 400,
+                            duration: 0
+                        }
+                    }
+                },
+
+                yAxis: {
+                    stackLabels: {
+                        enabled: true
+                    }
+                },
+
+                series: [{
+                    data: [43934, 52503, 57177, 69658, 97031, 119931, 137133, 154175]
+
+                }, {
+                    data: [43934, 52503, 57177, 69658, 97031, 119931, 137133, 154175].reverse()
+                }]
+            }),
+            done = assert.async(),
+            slOpacity;
+
+        setTimeout(function () {
+            // animation started
+            slOpacity = chart.yAxis[0].stackTotalGroup.opacity;
+            assert.strictEqual(
+                slOpacity === 0,
+                true,
```

Comment:
Instead of 
```
            assert.strictEqual(
                slOpacity === 0,
                true,
```

Use
```
            assert.strictEqual(
                slOpacity,
                0,
```

+ change `slOpacity` variable name to something more readable :)